autodiff/reverse/var/var.hpp

```diff
@@ -67,72 +69,127 @@ using std::sinh;
 using std::sqrt;
 using std::tan;
 using std::tanh;
-using std::hypot;
 
-template<typename T> struct Expr;
```

Comment:
Please keep these declarations on a single line. It's much more readable IMO the previous way.